docs/src/components/pricing/PricingTable.tsx

```diff
@@ -879,7 +879,7 @@ export default function PricingTable({
       {renderRow('@mui/system')}
 
       <RowHead startIcon={<IconImage name="product-advanced" width="28" height="28" />}>
-        Advanced
+        X Pro
```

Comment:
Same feedback as above, mentioning Pro feels out-of place in this context. Could it alienate the MIT use case?

Reply:
I'm going to resolve this one just so we focus the discussion on the already opened thread above. I'll change all instances of it once we settle on something!